tools/windows_portable/build_portable.ps1

```
@@ -351,18 +425,58 @@ function Assert-BundleRuntime([string]$BundleRoot) {
-  & $python -c "import tomli_w; import typer; import rich; import frame_compare"
+  & $python -c "import tomli_w; import typer; import rich; import vspreview; import PySide6; import frame_compare"
   Assert-LastExitCode -CommandLabel "bundle runtime import validation"
 }
 
+function Copy-PythonDistLicenses([string]$SitePackages, [string]$LicensesPythonDir) {
+  Ensure-Directory -Path $LicensesPythonDir
+  $licensePatterns = @("LICENSE*", "COPYING*")
+  $distInfoDirs = @(Get-ChildItem -LiteralPath $SitePackages -Directory -Filter "*.dist-info" -ErrorAction SilentlyContinue)
+  foreach ($distInfo in $distInfoDirs) {
+    foreach ($pattern in $licensePatterns) {
+      $matches = @(Get-ChildItem -LiteralPath $distInfo.FullName -File -Filter $pattern -ErrorAction SilentlyContinue)
```

PSScriptAnalyzer (1.24.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)